app/views/obituaries/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_for @obituary do |f| %>
```

Comment:
Added this form as a partial so we can reuse it in the edit form.

Reply:
Awesome, thank for doing that!

Reply:
Nice, this is awesome!

Reply:
Nice work here